marconi-sidechain/doc/API.adoc

```diff
@@ -0,0 +1,884 @@
+= JSON-RPC API
+:toc: left
+:toclevels: 2
+
+== HTTP endpoint
+
+In this application, there is a single HTTP endpoing/route: `POST /json-rpc`.
+This endpoint will respond to well-formed JSON-RPC requests.
```

Comment:
should the JSON-RPC be a link to the Spec? https://www.jsonrpc.org/specification
and the version we're using? 2.0